src/object/obj_internal.h

```diff
@@ -474,9 +474,10 @@ struct obj_auxi_args {
 					 rebuilding:1,
 					 for_migrate:1;
 	/* request flags. currently only: ORF_RESEND */
-	uint32_t			 flags;
 	uint32_t			 specified_shard;
+	uint16_t			 flags;
 	uint16_t			 retry_cnt;
+	uint16_t			 csum_retry_cnt;
```

Comment:
maybe we can directly re-use retry_cnt?

Reply:
the issue is that this counter can be incremented for many reason and we might end up not retrying on checksum error if the RPCs failed many time previously for another reason ...